app/models/miq_queue.rb

```diff
@@ -32,24 +32,22 @@ class MiqQueue < ApplicationRecord
   PRIORITY_WHICH  = [:max, :high, :normal, :low, :min]
   PRIORITY_DIR    = [:higher, :lower]
 
-  def self.artemis_client(client_ref)
-    @artemis_client ||= {}
-    @artemis_client[client_ref] ||= begin
+  def self.queue_type
+    ENV["QUEUE_TYPE"] || Settings.prototype.queue_type
+  end
+
+  def self.queue_client(client_ref)
+    @queue_client ||= {}
+    return if queue_type == "miq_queue"
+
+    @queue_client[client_ref] ||= begin
       require "manageiq-messaging"
       ManageIQ::Messaging.logger = _log
-      queue_settings = Settings.prototype.artemis
-      connect_opts = {
-        :host       => ENV["ARTEMIS_QUEUE_HOSTNAME"] || queue_settings.queue_hostname,
```

Comment:
TODO rename to MESSAGING_HOSTNAME/MESSAGING_PORT/etc...

Reply:
:confused: Do you want to rename those for this PR? Should I go with the new names for the pods PRs?

Reply:
@carbonin yes I just updated them in this PR